modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -39,11 +39,24 @@ For more information about how to install Helm, see the Helm installation docume
 . Add and list ValidatingAdmissionWebhook admission controllers in the correct order.
 
 
+== Hardware Requirements
+
+Your Anypoint Service Mesh installation requires the following hardware configuration:
+
+* CPU: 4 cores
+* Memory: 8 GBs
+
+
 == Production Licensing and Trial Licensing Requirements
 
-You require a Anypoint Service Mesh license to perform the installation in your production environment.
+You require a Anypoint Service Mesh license to perform the installation in your production environment. Ensure that the license you specify is in a digested format. If you do not have a license, contact your MuleSoft representative.
```

Comment:
> You require **an** Anypoint...

"digested format", is the user aware of what digested format means? I'm just asking because I don't know

Reply:
IMHO, we shouldn't spend much time on this as we will probably support the other type of license soon.

Reply:
Done! Thanks!